newrelic/common/agent_http.py

```diff
@@ -119,6 +123,11 @@ def log_request(cls, fp, method, url, params, payload, headers, body=None, compr
         if not fp:
             return
 
+        # Obfuscate license key from params
```

Comment:
The API key is part of the headers in the OTLP class and the NewRelicLogHandler (See line 557 in newrelic/core/agent_protocol.py). I believe those get passed into this function call as headers and get printed out to the file below on line 149 so we need to obfuscate that too.

Reply:
You're right, the headers elsewhere get filtered but not in the audit log. I added exclusions for those 2 cases and the `record_deploy` command which uses `X-Api-Key`.